src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +26,13 @@ public interface MachineService {
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super T> machines, T value);
```

Comment:
Ensure that the size of the list passed to `fill()` method remains the same after the method execution. Replace the old values with the new ones without changing the list size.